.github/actions/spelling/excludes.txt

```diff
@@ -54,14 +54,26 @@ SUMS$
 \.zip$
 ^consolegit2gitfilters\.json$
 ^dep/
-^oss/
+^doc/reference/master-sequence-list.csv$
 ^doc/reference/UTF8-torture-test\.txt$
+^oss/
+^src/host/ft_uia/run\.bat$
+^src/host/runft\.bat$
+^src/host/runut\.bat$
```

Comment:
These files (and the others above/below) were deemed by the tool as having fewer words than unknown words. If any of them should be flagged for typos, then some of the unknown words would need to be added to the dictionary, or strings would need to be masked until the balance is more words than unknown tokens.